modules/RTC/RTC.js

```diff
@@ -11,11 +11,12 @@ var DesktopSharingEventTypes
 var MediaStreamType = require("../../service/RTC/MediaStreamTypes");
 var RTCEvents = require("../../service/RTC/RTCEvents.js");
 
-function createLocalTracks(streams) {
+function createLocalTracks(streams, options) {
     var newStreams = []
     for (var i = 0; i < streams.length; i++) {
+        var deviceId = options.cameraDeviceId || options.micDeviceId;
```

Comment:
Shouldn't we check the type of the stream here and pass options.cameraDeviceId for video streams, options.micDeviceId for audio and null for desktop streams?